scripts/gcs-cluster-ui

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+##############################################################
+#  Script to easily open a gcs dataproc cluster's spark UI   #
```

Comment:
I think the script is slightly misnamed, since GCS = "Google Cloud Storage". Maybe `gcp-dataproc-cluster-ui` would be more accurate?

Reply:
that's a good point, should we rename it in other places like the GATK launcher too?